generic/tests/linux_stress.py

```diff
@@ -0,0 +1,203 @@
+import time
+import logging
+import aexpect
+
+from avocado.utils import download
+
+from virttest import data_dir
+from virttest import utils_misc
+from virttest import error_context
+
+
+class Stress(object):
+    """
+    Stress function for linux guest
+    """
+
+    def __init__(self, params, env):
```

Comment:
Miss "test" here

Reply:
@huangyum This class does not need `test` object. Thanks.

Reply:
I'm not sure.  But I was told to add "test","params","env" for all test classes.

Reply:
@huangyum Yes, I got that idea from maintainer too, but at same time there is another type class, which is not a "test" class, but a functional class, so does this one. It only provide functions, it will not decide whether the test pass or fail, so do not need `test` module.